### PR TITLE
Fixes #146: test case added for util

### LINKED ISSUE #146: Add test case for util
### This is a
- [ ] Bug report
- [x] Feature request

### Current behaviour

no test case for util decrypter file
### Expected behaviour

add test case for util decrypter file and make the test to run on every travis build
### Steps to reproduce
1.
2.
3.

### PR DESCRIPTION
**Pull request by sch00lb0y**
## Description

test case addded for util decrypter.js
## How Has This Been Tested?

Tested in travis. here is the [travis log](https://travis-ci.org/sch00lb0y/yaydoc/builds/244423452) and I have updated the tested travis screenshot
## Screenshots (if appropriate):
travis-decrypt-test
## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by sch00lb0y** [comment]
@imujjwal96 @pri22296 please review this

File: .travis.yml [hunk 1]
```diff
@@ -7,12 +9,18 @@ env:
     - DOCURL=yaydoc.fossasia.org
     - PROJECTNAME=Yaydoc
     - VERSION=development
+    - ENCRYPTION_KEY=ABCGDHEUSHFKIRMJSDFIUTEHFNGOWPOR
```
Comment (pri22296) [inline .travis.yml]: where is this used?

Comment (sch00lb0y) [inline .travis.yml]: yeah, we not using this key in production. so then it is safe

Comment (pri22296) [inline .travis.yml]: Ok den, I'm approving this

File: .travis.yml [hunk 2]
```diff
@@ -7,12 +9,18 @@ env:
...
+
+install:
+  - npm install -g mocha
```
Comment (imujjwal96) [inline .travis.yml]: Add mocha to `package.json` and remove it from here.

Comment (sch00lb0y) [inline .travis.yml]: @imujjwal96 I have made changes as you suggested

**Pull request merged**